lib/internal/bootstrap_node.js

```diff
@@ -182,7 +182,7 @@
     process._setupProcessObject(pushValueToArray);
 
     function pushValueToArray() {
-      for (var i = 0; i < arguments.length; i++)
+      for (let i = 0; i < arguments.length; i++)
```

Comment:
This is used via a few c++ functions to improve the performance of pushing onto an array. I'm pretty sure `let` is still a bit slower than `var`, so I would probably leave this one alone